GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -304,17 +283,14 @@ - (void)setUp {
   
   _httpFetcher = [[GIDFakeHTTPFetcher alloc] init];
   
-  _profileDataFetcher = [[GIDFakeProfileDataFetcher alloc] init];
+  _authorizationFlowProcessor = [[GIDFakeAuthorizationFlowProcessor alloc] init];
   
-  GIDAuthorizationFlowProcessor * authorizationFlowProcessor =
-      [[GIDAuthorizationFlowProcessor alloc] init];
+  _profileDataFetcher = [[GIDFakeProfileDataFetcher alloc] init];
```

Comment:
Swap these two lines to match the order of variables being passed to GIDSignIn's initializer below.